packages/api-server/src/db/query.ts

```diff
@@ -316,7 +316,7 @@ export class Query {
       .where("block_number", ">=", fromBlock)
       .where("block_number", "<=", toBlock)
       .where("id", ">", queryLastPollId.toString(10))
-      .orderBy("id", "desc")
+      .orderBy("id", "asc")
```

Comment:
why order by asc?

Reply:
- https://github.com/nervosnetwork/godwoken-web3/issues/273

Other chains seems order from lower blocks to higher blocks

Reply:
Thanks, it makes sense to me.